src/appsignal/cli.py

```diff
@@ -152,6 +148,74 @@ def _write_file(self):
 
             f.write(file_contents)
 
+    def _requirements_file(self):
+        current_dir = os.getcwd()
+        for _root, _dirs, files in os.walk(current_dir):
+            for file in files:
+                if file.endswith("requirements.txt"):
+                    return file
+        return None
+
+    def _search_dependency(self, dependency_name):
+        requirement_file = self._requirements_file()
+        if requirement_file:
+            with open(requirement_file, "r") as f:
+                for line in f.readlines():
+                    if line.startswith(dependency_name):
+                        return True
+                    else:
+                        return False
+
+        return False
+
+    def _django_installation(self):
+        print("We've detected that you're using Django.")
+        print()
+
+        if not self._search_dependency("opentelemetry-instrumentation-django"):
+            print("Adding the Django instrumentation to your requirements.txt file")
+            print()
+            self._add_dependency("opentelemetry-instrumentation-django")
+
+        print("Django requires some manual configuration.")
+        print("The __appsignal__ module needs to be imported in the manage.py file")
+        print("and the appsignal.start() method needs to be called in the main method.")
+        print()
+        print("Please refer to the documentation for more information:")
+        print("https://docs.appsignal.com/python/instrumentations/django.html")
+
+    def _flask_installation(self):
+        print("We've detected that you're using Flask.")
+        print()
+
+        if not self._search_dependency("opentelemetry-instrumentation-flask"):
+            print("Adding the Flask instrumentation to your requirements.txt file")
+            print()
+            self._add_dependency("opentelemetry-instrumentation-flask")
+
+        print("Flask requires some manual configuration.")
+        print("The __appsignal__ module needs to be imported before Flask is imported")
+        print("and the appsignal.start() method needs to be called right after.")
+        print()
+        print("Please refer to the documentation for more information:")
+        print("https://docs.appsignal.com/python/instrumentations/flask.html")
+
+    def _generic_installation(self):
+        print("✅ Done! AppSignal for Python has now been installed.")
+        print()
+        print("To start AppSignal in your application, add the following code to your")
+        print("application's entrypoint:")
+        print()
+        print("    from __appsignal__ import appsignal")
+        print("    appsignal.start()")
+        print()
```

Comment:
This branch of the installation could also use a documentation link. Maybe to the list of instrumentations?